configure

```diff
@@ -2169,7 +2193,7 @@ if test -n "$CXX" && test_compiler "$CXX"; then
 	# Use the compiler specified in CXX
 	echo $CXX
 else
-	if test -n "$_host"; then
+	if test -n "$_host" && test "$_host" != sailfish ; then
```

Comment:
Could you add a comment explaining this bit?

Reply:
Done